src/Student.java

```diff
@@ -0,0 +1,58 @@
+import java.lang.invoke.SwitchPoint;
+
+public class Student {
+    private String name;
+    private  Characteristics[] characteristics;
+    double magicPower;
```

Comment:
Свойства должны быть приватными
```suggestion
    private double magicPower;
```